main.go

```diff
@@ -1,17 +1,91 @@
 package main
 
 import (
+	"fmt"
+	"io"
 	"log"
+	"log/slog"
 	"os"
+	"path"
+	"strings"
+	"time"
 
 	"github.com/joshmedeski/sesh/seshcli"
 )
 
 var version = "dev"
 
 func main() {
+	slog.Debug("Debug")
+	slog.Info("Information")
+	slog.Warn("Warning")
+	slog.Error("Error")
+
 	app := seshcli.App(version)
 	if err := app.Run(os.Args); err != nil {
 		log.Fatal(err)
 	}
 }
+
+func init() {
+	var f *os.File
+	var err error
+	fileOnly := false
+
+	if f, err = createLoggerFile(); err != nil {
+		log.Fatalf("Unable to create logger file: %v", err)
+	}
+	if f == nil {
+		log.Println("init: no file")
+	}
+
+	env := os.Getenv("ENV")
+
+	handlerOptions := &slog.HandlerOptions{}
+
+	switch strings.ToLower(env) {
+	case "debug":
+		handlerOptions.Level = slog.LevelDebug
+	case "info":
+		handlerOptions.Level = slog.LevelInfo
+	case "error":
+		handlerOptions.Level = slog.LevelError
+	default:
+		handlerOptions.Level = slog.LevelWarn
+		fileOnly = true
+	}
+
+	var loggerHandler *slog.JSONHandler
+	if !fileOnly {
+		multiWriter := io.MultiWriter(os.Stdout, f)
+		loggerHandler = slog.NewJSONHandler(multiWriter, handlerOptions)
+	} else {
+		loggerHandler = slog.NewJSONHandler(f, handlerOptions)
+	}
+	slog.SetDefault(slog.New(loggerHandler))
+}
+
+func createLoggerFile() (*os.File, error) {
+	now := time.Now()
+	date := fmt.Sprintf("%s.log", now.Format("2006-01-02"))
+	userHomeDir, err := os.UserHomeDir()
+	if err != nil {
+		return nil, err
+	}
+
+	if err := os.MkdirAll(path.Join(userHomeDir, ".config", "sesh"), 0755); err != nil {
```

Comment:
I'd prefer we put this in the OS's temp directory. I wouldn't want people who sync `.config/sesh` with a repo to have to deal with ignoring log files.

Reply:
Defaulted to the OS's temp directory, inside of which a `sesh` directory will be created where all the log files will be saved to, agreed with you that having the log files in the `.config/sesh` directory wasn't the best choice